core/src/main/scala/akka/kafka/scaladsl/Processor.scala

```diff
@@ -0,0 +1,44 @@
+package akka.kafka.scaladsl
+
+import akka.kafka.ProducerMessage.Envelope
+import akka.kafka.scaladsl.Consumer.DrainingControl
+import akka.kafka.{CommitterSettings, ConsumerSettings, ProducerSettings, Subscription}
+import akka.stream.Materializer
+import akka.stream.scaladsl.Keep
+import akka.{Done, NotUsed}
+import org.apache.kafka.clients.consumer.ConsumerRecord
+
+import scala.concurrent.Future
+
+object Processor {
+
+  def atLeastOnce[K, V](
```

Comment:
What about adding this method to `Consumer` instead? 

I find it odd that we have `Consumer.atMostOnce`, but the `atLeastOnce` is on another object.

Reply:
`Consumer` is very overloaded already. But it might end up there anyway.